cmd/crank/beta/render/cmd.go

```diff
@@ -206,3 +211,13 @@ func (c *Cmd) Run(k *kong.Context, _ logging.Logger) error { //nolint:gocyclo //
 
 	return nil
 }
+
+// sortByKindAndName sorts a slice of composed resources in place, first by GroupVersionKind and then by name.
+func sortComposedResources(out []composed.Unstructured) {
+	sort.Slice(out, func(i, j int) bool {
+		if comp := strings.Compare(out[i].GroupVersionKind().String(), out[j].GroupVersionKind().String()); comp != 0 {
+			return comp < 0
+		}
+		return strings.Compare(out[i].GetName(), out[j].GetName()) < 0
```

Comment:
I notice the godoc for `strings.Compare` says:

> Compare is included only for symmetry with package bytes. It is usually clearer and always faster to use the built-in string comparison operators ==, <, >, and so on.

```suggestion
		return out[i].GroupVersionKind.String() + out[i].GetName() < out[j].GroupVersionKind.String() + out[j].GetName()
```

I'd find this more intuitive to read, personally.